extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketConnectorImpl.java

```diff
@@ -63,6 +66,12 @@ public Uni<WebSocketClientConnection> connect() {
             clientOptions.setMaxMessageSize(config.maxMessageSize().getAsInt());
         }
 
+        Optional<TlsConfiguration> tlsConfiguration = TlsConfiguration.from(tlsConfigurationRegistry,
+                config.tlsConfigurationName());
+        if (tlsConfiguration.isPresent()) {
+            clientOptions.setTrustOptions(tlsConfiguration.get().getTrustStoreOptions());
```

Comment:
There is a bit more to do:

- alpn, handshake timeout, trust-all...

Reply:
I will add them tomorrow

Reply:
Added